### DISCUSSION TITLE
In which cases page have title and subtitle?

### DISCUSSION THREAD

**Discussion by sheldygg** [aminalaee/sqladmin#856] [Q&A]
<img width="1421" alt="image" src="https://github.com/user-attachments/assets/ccde0830-3f6c-4dc1-bc15-b2916c67a71e">
I see this element on page and he always empty, when it will be used?

**Comment by mmzeynalli** [comment]
It is redundant right now. And we have h3 for model name, so we can remove that. Thanks for mentioning!
